### PR TITLE
[offsets-tool] Remove the obsolete '--maccore=' argument, its not used anymore.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: tools/offsets-tool/MonoAotOffsetsDumper.cs [hunk 1]
```diff
@@ -368,14 +360,11 @@ static void SetupMono(Driver driver, Target target)
...
-                    string targetPath = Path.Combine (MaccoreDir, "builds");
-                    if (!Directory.Exists (MonoDir))
-                        MonoDir = Path.GetFullPath (Path.Combine (targetPath, "../../mono"));
-                    targetBuild = Path.Combine(targetPath, target.Build);
-                }
+                if (string.IsNullOrEmpty (TargetDir)) {
+                    Console.Error.WriteLine ("The --targetdir= option is required when targeting ios.");
+					Environment.Exit (1);
+				}
+				targetBuild = TargetDir;
```
Comment (lewurm) [inline tools/offsets-tool/MonoAotOffsetsDumper.cs]: indent 🙂

**Comment by vargaz** [comment]
@monojenkins build Linux x64 Interpreter

**Comment by vargaz** [comment]
@monojenkins build Linux x64 Interpreter

**Comment by vargaz** [comment]
@monojenkins build API Diff

**Comment by vargaz** [comment]
@monojenkins build Project Files Diff

**Pull request merged**